app/templates/events/live.hbs

```diff
@@ -1 +1 @@
-{{events/events-table columns=columns data=data style='margin-top: 20px;'}}
```

Comment:
@geekyd Hi Thanks! @niranjan94 had suggested to remove this. you can check outdated comments